src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -11,53 +11,81 @@ Note that because k6 does not run in NodeJS, the browser module APIs will slight
 
-### Devices Example
 
-To emulate the browser behaviour on a mobile device and approximately measure the browser performance, you can import `devices` from `k6/experimental/browser`.
+## Browser Module API
 
-<CodeGroup labels={["script.js"]} lineNumbers={[true]}>
+The browser module is the entry point for all your tests, and it is what interacts with the actual web browser via [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) (CDP). It manages:
+- [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) which is where you can set a variety of attributes to control the behavior of pages;
+- and [Page](/javascript-api/k6-experimental/browser/page/) which is where your rendered site is displayed.
+
+A new Chromium browser process is automatically launched when you use the `newContext` method of the `browser` module (`'k6/experimental/browser'`).
+
+| Method                                                                                    | Description                                                                                                                                           |
+|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [browser.close()](/javascript-api/k6-experimental/browser/close)                              | Closes the browser and all of its pages (if any were opened).                                                                                         |
+| [browser.contexts()](/javascript-api/k6-experimental/browser/contexts)                        | Lets you access all open [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/)s.                                                        |
+| [browser.isConnected](/javascript-api/k6-experimental/browser/isconnected) <BWIPT id="453"/>  | Indicates whether the [CDP](https://chromedevtools.github.io/devtools-protocol/) connection to the browser process is active or not.                  |
+| [browser.newContext([options])](/javascript-api/k6-experimental/browser/newcontext/) <BWIPT id="455"/> | Creates and returns a new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/).                                                             |
+| [browser.newPage([options])](/javascript-api/k6-experimental/browser/newpage)  <BWIPT id="455"/>       | Creates a new [Page](/javascript-api/k6-experimental/browser/page/) in a new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) and returns the page. |
+| [browser.on('disconnected')](/javascript-api/k6-experimental/browser/on/) | Detects the disconnected event from the browser application. |
```

Comment:
I removed this in https://github.com/grafana/k6-docs/pull/1197. Be aware of conflicts.

Reply:
Also removed here 👍